README.md

```diff
@@ -170,6 +170,38 @@ jobs:
 
 </details>
 
+### Using pre-commit
+[pre-commit](https://pre-commit.com/) lets you easily install and manage pre-commit hooks.
+
+First, install pre-commit. Then, add the following to your `.pre-commit-config.yaml`
+
+```yaml
+-   repo: <link-to-repo>
```

Comment:
Should a user link to this repository? If so, then we should make this example copy-paste friendly.

If not, I'm not sure I quite understand what a user should use in this field.